querydsl-hibernate-search/pom.xml

```diff
@@ -72,9 +72,9 @@
     </dependency>    
  
     <dependency>
-      <groupId>org.hibernate.javax.persistence</groupId>
-      <artifactId>hibernate-jpa-2.0-api</artifactId>
-      <version>1.0.0.Final</version>
+      <groupId>jakarta.persistence</groupId>
+      <artifactId>jakarta.persistence-api</artifactId>
+      <version>2.2.3</version>
       <scope>compile</scope>
```

Comment:
I think this should be `provided` scope as well, equivalently to `querydsl-jpa`. Eventually it will be the used ORM implementation that provides the supported JPA API transitively.

Reply:
Changed to `provided`.